### DISCUSSION TITLE
dependencyDashboardLabels overwrites manual label changes in GitLab

### DISCUSSION THREAD

**Discussion by Boojapho** [renovatebot/renovate#11888] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

25.56.1

### Describe the bug

When running with the Dependency Dashboard, we set the following option:
`dependencyDashboardLabels: ["priority::3","type::chore"]`.  The creation of the dashboard issue in GitLab works and the labels specified are applied.

Now, if someone manually changes the labels on the issue in GitLab to `priority::1` and adds another label `emergency`, the next time renovate runs, it will reset the label to `priority::3` and remove `emergency`.  This only happens on the Dependency Dashboard and not on the MR.

I would expect that labels are only applied when the issue is Opened (either via creation or re-opening) and that the labels are left alone when updating an open dashboard.

[Minimum Reproduction Repository](https://gitlab.com/Boojapho/renovate)

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by viceice** [comment, marked as answer]
This is currently by design

**Reply by rarkins** [reply]
Yes, this is by design and documented. The reasoning is:
- Dashboard labels are optional. Leave them empty if you want flexibility
- When this feature was introduced many people had *existing* dashboards. Dashboards aren't created all the time like PRs are - e.g. the one in this repo has been open since 2018. So the decision was to add labels to any existing issue if enabled

Now that this feature has been rolled out for a while and everyone's dashboard issues should be labeled if desired, we could consider switching this back to "only on create" in the next major release.

**Reply by viceice** [reply]
Sounds good
